sympy/matrices/expressions/tests/test_matexpr.py

```diff
@@ -76,6 +76,40 @@ def test_ZeroMatrix_doit():
     assert isinstance(Znn.doit().rows, Mul)
 
 
+def test_OneMatrix():
+    A = MatrixSymbol('A', n, m)
+    a = MatrixSymbol('a', n, 1)
+    U = OneMatrix(n, m)
+
+    assert U.shape == (n, m)
+
+    assert isinstance(A + U, Add)
+    # assert a*OneMatrix(n, 1) == a
+    # assert OneMatrix(n, 1)*a == a
+
+    assert transpose(U) == OneMatrix(m, n)
+    assert U.conjugate() == U
+
+    assert OneMatrix(n, n) ** 0 == Identity(n)
+    with raises(ShapeError):
+        U ** 0
+    with raises(ShapeError):
+        U ** 2
+
+    U = OneMatrix(n, n)
+    assert U[1, 2] == 1
```

Comment:
I am not sure that matrices with symbolic shape can have its elements defined at explicit positions.

Reply:
`ZeroMatrix` behaves like this (always returns zero). There should be a bounds check, but I don't think this is important. If you access a matrix containing only one element, you're pretty sure you get that element.